cardano-node/Cardano/Node/CLI.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RecordWildCards     #-}
```

Comment:
Spacing here is a bit odd

Reply:
Please avoid using `RecordWIldCards`. In almost all circumstances it makes code less clear, readable and maintainable.

Reply:
Dropped RWC & fixed spacing.